index.bs

```diff
@@ -771,6 +764,23 @@ Note: For example, the UA may estimate [=optimal sampling frequency=] as a Least
 (LCD) for a set of {{[[desiredSamplingFrequency]]|desired sampling frequencies}} capped by
 [=sampling frequency=] bounds defined by the underlying platform.
 
+<h3 id="model-sensor">Conditions to update latest reading</h3>
```

Comment:
maybe we could convert it to a feature of the browsing context and move this section into "Concepts"? For example:
"The browsing context is \<dfn\>safe to share platform data\<\/dfn\> if the following conditions are met ..." and in the text above it could be "Any time a new [=sensor reading=] for a [=platform sensor=] is obtained and if the current browsing context is [=safe to share platform data=] the user agent invokes [=update latest reading=]..." WDYT?

Reply:
Sounds good. `<dfn>safe to provide sensor data</dfn>`

platform => sensor. To make it explicit, platform has lots of other data
share => provide.  Share might sound as if UA shares data with something other than web page.

wdyt?

Reply:
I'm fine with that, maybe "propagate" would be even more appropriate verb here: "safe to propagate sensor readings (to be consistent :-) )"

Reply:
Alternative proposals for the definition:
`<dfn>can expose sensor readings</dfn>`
`<dfn>check if we can expose sensor readings</dfn>`

Where "expose" refers to "expose to scripts" specifically.

To be consistent with e.g.:

https://html.spec.whatwg.org/#can-share-memory-with
https://html.spec.whatwg.org/#check-if-we-can-run-script

The term "propagate" has its own meaning in https://dom.spec.whatwg.org/ so I'd rather avoid it unless we define it more specifically.

Reply:
- Added `<dfn>can expose sensor readings</dfn>`
- Moved section to concepts